packages/cli/src/lib/cli.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { execSync } from 'child_process';
+import { cli } from './cli';
+
+describe('cli', () => {
+  beforeAll(() => {
+    // symlink NPM workspaces
+    execSync('npm install');
```

Comment:
Dont get this, why are you syncing the npm in the test

Reply:
The config files used in the test have imports from the NPM package names (`@quality-metrics/eslint-plugin` and `@quality-metrics/lighthouse-plugin`). This works via symlinks which NPM creates automatically when installing thanks to the `"workspaces": "dist/packages/*"` in `package.json`.

The problem is that, in the CI job, `npm install` runs before the project `build`s, so the symlinks don't get created because `dist` is empty at that time (less common locally), so [the test fails](https://github.com/flowup/quality-metrics-cli/actions/runs/6011150937/job/16303950914). So I re-run `npm install` here just so that NPM creates the symlinks and the test works.

I'd like address this in a better way once we move out these E2E tests in #19, but for now I just wanted a passing CI and not to do too much in one PR.